supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -168,6 +168,26 @@ If referring to the program, use "Emacs", for example, "Source-Navigator support
 
 *See also*:
 
+[[executable]]
+==== image:images/yes.png[yes] Executable (adjective)
+*Description*: _Executable_ describes a file an operating system can run as a program. For directories, having executable permission allows you to navigate into the directory with the `cd` command.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[[executable-file]]
+==== image:images/yes.png[yes] Executable file (noun)
+*Description*: When it runs, an _executable file_ or _executable program_ performs specific operations based on its coded instructions. Write _executable file_ on the first use and, optionally, _executable_ thereafter.
```

Comment:
I think when @bredamc proposed this, she proposed putting "executable" in parentheses after the first use of "executable program".

Should we also consider adding a separate entry for "executable" as a noun?